float32 goal_x
float32 goal_y
float32 goal_orientation
bool result_of_navigation
